lib/govuk_component/shared_helpers.rb

```diff
@@ -10,5 +10,9 @@ def prepend_classes(classes_to_prepend, options = {})
       options[:class] = classes.join(' ')
       options
     end
+
+    def capture_or_arg(data = nil, &block)
+      block ? capture(&block) : data
```

Comment:
use `block_given?` as block will call the block which can have side effects
```suggestion
block_given? ? capture(&block) : data
```

Reply:
This is outdated as the file no longer exists, however, `block_given?` generates a Rubocop offense: https://msp-greg.github.io/rubocop-performance/RuboCop/Cop/Performance/BlockGivenWithExplicitBlock.html

Reply:
> block_given? generates a Rubocop offence

I was going to suggest not passing named block and using `yield` instead